src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/LicencesAndRights/LegacyConfig/PlayerConfigProvider.cs

```diff
@@ -118,7 +118,7 @@ private SimpleDialogue GetHelpDialogue()
             dict["OPL"] = "You have permission to make copies of this work under an <a target=\"_top\" href=\"http://www.parliament.uk/site-information/copyright/open-parliament-licence/\">Open Parliament license</a>.<br/><br/>This licence permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited. <br/><br/>Image source should be attributed as specified in the full catalogue record. If no source is given the image should be attributed to Wellcome Library.";
             dict["ARR"] = "The work has been made available under an \"all rights reserved licence\". See the full catalogue record for further information about what rights you have to make copies of this work.";
             dict["Copyright not cleared"] = "The copyright of this item has not been evaluated. Please refer to the original publisher/creator of this item for more information. You are free to use this item in any way that is permitted by the copyright and related rights legislation that applies to your use. <br/>See <a target=\"_top\" href=\"http://rightsstatements.org/page/CNE/1.0/?language=en\">rightsstatements.org</a> for more information.";
-            dict["In copyright"] = "Conditions of use: it is possible this item is protected by copyright and/or related rights. You are free to use this item in any way that is permitted by the copyright and related rights legislation that applies to your use. For other uses you need to obtain permission from the rights-holder(s).";
+            dict[Constants.InCopyrightCondition] = Constants.InCopyrightConditionStatement;
```

Comment:
I suspect you;ll find a few where you'll need to do exactly the same with the "Copyright not cleared" above this one.

Reply:
Added handling for "Copyright not cleared"